spec/data/acceptance/css/u4p1.css

```diff
@@ -3,36 +3,36 @@ body{
     background-size:cover;
     color:silver;
     }
-    #banner{
-        position:relative;
-        top:20px;
-        left:250px;
-        }
-        #tagline{
-            position:relative;
-            top:600px;
-            left:250px;
-            }
-            img{
-            position:relative;
-            top:10px;
-            left:10px;
-            height:100px;
-            width:500px;
-            }
-            #mission{
-            position:relative;
-            top:300px;
-            left:100px;
-            }
-            #about{
-            position:relative;
-            top:100px;
-            left:300px;
-            }
-            #topics{
-            position:relative;
-            top:300px;
-            left:400px;
-            }
-    
\ No newline at end of file
+#banner{
+    position:relative;
+    top:20px;
+    left:250px;
+}
+#tagline{
+    position:relative;
+    top:600px;
+    left:250px;
+}
+img{
+    position:relative;
+    top:10px;
+    left:10px;
+    height:100px;
+    width:500px;
+    filter: contrast(175%) brightness(3%);
+}
+#mission{
+    position:relative;
+    top:300px;
+    left:100px;
+}
+#about{
+    position:relative;
+    top:100px;
+    left:300px;
+}
+#topics{
+    position:relative;
+    top:300px;
+    left:400px;
+}
```

Comment:
This isn’t at all clear, but the stuff in `spec/data/acceptance` is lifted verbatim from the ScriptEd curriculum; the idea is to make sure that changes to the validations don’t inadvertently break something that ScriptEd thinks of as valid.

But I’d love it if you would add a new test case to the [CSS spec](https://github.com/popcodeorg/popcode/blob/master/spec/examples/validations/css.spec.js)!